docs/source/python_api/mlflow.models.evaluation.base.rst

```diff
@@ -1,7 +0,0 @@
-mlflow.models.evaluation.base
```

Comment:
@WeichenXu123 Can we also delete https://github.com/mlflow/mlflow/blob/master/docs/source/python_api/mlflow.models.evaluation.rst, since `mlflow.models` includes everything that we want to document?